src/helpers.js

```diff
@@ -0,0 +1,21 @@
+export const People = {
```

Comment:
Yep, but it should be lowercased :) 

In react, you use camelCase with uppercased first letter only for components.